next/content/math-mode.md

```diff
@@ -0,0 +1,432 @@
+---
+title: Math Mode
+bookhidden: true
+math-mode: true
+---
+
+# Introduction
+---
+{{<plain hidden>}}
```

Comment:
That's not really needed. Without this shorttag it will put some dom there, but it will only display as an empty paragraph, which I think is good enough.

Without a shortcode it will be valid Markdown if you strip off the header.

Reply:
strip off the header ?
this is needed so we dont get the flash of math code

Reply:
Stripping of the header is what I do if I render without Hugo (though I can see that it's not a priority and I can see that we don't really need it).

The flashing is a good point, other than that I don't see the need of using a special shortcode (but again I've fine with it). I just wanted to bring it up as less shortcodes also makes it easier for authors.

Reply:
what about the `\_` ?
fyi in the future i will add a pipeline to make this static

Reply:
You just put it into a code block, it will be hidden by KaTeX automatically as doesn't contain any data. The only artifact will be a bit of space as it renders it into a `<span>`.

Reply:
Oh wait, I can even put it into the first actual code block, that would work as well. No special short codes would be needed. I can just do that an show it to you in my PR and then we can decide :)

Reply:
code blocks for hidden math dont work it renders an empty box